pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -675,6 +675,14 @@ func (dm *DockerManager) runContainer(
 			// TODO: Clean up the previously created dir? return the error?
 			glog.Errorf("Error on creating termination-log file %q: %v", containerLogPath, err)
 		} else {
+			// Termination log should be writable by the process group. This is
+			// especially important in the remap environment because otherwise
+			// it won't be possible to write the status. Docker could adjust
+			// permissions on this file for us but it means that right after
+			// container creation, file will be non-writable for a short period.
+			if err := fs.Chmod(0664); err != nil {
```

Comment:
so was the issue here that SyncPod was calling `ensureDirectoryAccess` prior to this file being created and then, on the subsequent Sync call it would update the permissions?  Do you think `ensureDirectoryAccess` be moved to a different place?  It seems like there is a possibility of it chowing something on an already started container now.  Ie. should it only be called in the `ContainersToStart` loop or possibly one of the called methods there like `tryStartContainer`?